docs/08-developer-guide/04-project/03-release-guide.md

```diff
@@ -4,6 +4,372 @@ sidebar_label: Release Manager Guide
+### Write an announcement mail to the Ozone mailing lists
+
+Include the following links:
+
+ * Release notes: [https://ozone.apache.org/release/1.2.0/](https://ozone.apache.org/release/1.2.0/). Replace the version in the URL with the version being released.
+ * Download link: [https://ozone.apache.org/downloads/](https://ozone.apache.org/downloads/)
+ * Link to versioned documentation: [https://ozone.apache.org/docs/](https://ozone.apache.org/docs/)
+
+### Publish a docker image for the release
+
+The Ozone docker image is intended for testing purposes only, not production use. Therefore, it is ok to update this after announcing the release. An example pull request to update the docker image is [here](https://github.com/apache/ozone-docker/pull/22/files). The target branch for your pull request should be `latest`. After the pull request is merged, it can be published to docker hub by fast-forwarding the `ozone-latest` branch to match the `latest` branch. Also, create a new branch named like `ozone-1.5.0` (replace the version) and push it to [GitHub](https://github.com/apache/ozone-docker).
+
+```bash
+git checkout ozone-latest
+git pull
+git merge --ff-only origin/latest
+git checkout -b "ozone-$VERSION"
+git push origin ozone-latest "ozone-$VERSION"
+```
+
+## Patch Release
+
+If there is a security vulnerability or critical bug uncovered in a major or minor Ozone release, a patch release may be necessary to fix this. The process is a bit simpler than a major or minor release, since there is already a solid foundation on the release's maintenance branch.
+
+ 1. Cherry pick the fix(es) on to the maintenance branch. For example, for Ozone's 1.2.0 release, this is the branch called ozone-1.2.
+ 2. Run all steps from the sections **Update the versions** through **Publish a docker image for the release**, with the following modifications:
+    * Do not update the protolock files unless protocol buffers were changed as part of the fix.
+    * When updating the website, all instances of the original major/minor release should be replaced with this patch version, since we do not want users downloading the original release anymore.
+      * For example, any website text referring to 1.2.0 should be changed to refer to 1.2.1.
+      * Continuing the 1.2.0 &rarr; 1.2.1 example, the release/1.2.0 page should redirect to release/1.2.1.
+      * An example pull request to do this is [here](https://github.com/apache/ozone-site/pull/23).
+      * The docs can be added to the website normally as described above in **Update the ozone website**. The docs link for the original major/minor release can remain alongside the docs link for the patch release.
+
+In the event of a critical security vulnerability or seriously harmful bug with a small set of changes in the patch, PMC members may vote to forgo the usual 72 hour minimum time for a release vote and publish once there are enough binding +1s.
+
+Remove the previous patch release from Apache distribution site:
```

Comment:
Making this part of the ordered list is fine, but I think we should keep the word "patch", or somehow emphasize that this applies to the same minor version release line, not just any previous release.

Reply:
I suggested removing "patch" because the previous release may not have been a patch release. For example 1.2.0 is not a patch release, but it would be removed in favor of 1.2.1. We can clarify the wording so that it refers to the same release line in the follow up PR.